tools/editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -1350,19 +1350,23 @@ AnimationPlayerEditor::AnimationPlayerEditor(EditorNode *p_editor) {
 
 
 	add_anim = memnew( ToolButton );
+	ED_SHORTCUT("animation_player_editor/add_animation", TTR("Create new animation in player."));
+	add_anim->set_shortcut(ED_GET_SHORTCUT("animation_player_editor/add_animation"));
```

Comment:
If it's more readable this way, then I think it's fine this way. I just wanted to be sure there was not something else I was missing :P